src/GovUk.Education.ExploreEducationStatistics.Data.Api/Controllers/TableBuilderController.cs

```diff
@@ -121,6 +123,25 @@ public async Task<ActionResult<TableBuilderResultViewModel>> QueryForTableBuilde
             return actionResult;
         }
 
+        [HttpGet("tablebuilder/release/data-block/{dataBlockParentId:guid}")]
```

Comment:
To make this convenient for the frontend code, this endpoint's path needs to align with its twin, minus the `data/`: `[HttpGet("data/tablebuilder/release/{releaseVersionId:guid}/data-block/{dataBlockParentId:guid}/geojson")]`

We need the releaseId for the Admin endpoint, so we'll also include it in the Data API endpoint path even though it isn't used. There is precedent for this - see the comment for the Data API's `TableBuilderController#QueryForTableBuilderResult`.